### PR TITLE
workflows/triage: auto-label `bump-formula-pr`

### LINKED ISSUE #76763: Label `brew bump-formula-pr` issues with `brew-bump-formula-pr` label
A `brew-bump-formula-pr` label if a PR's body contains `brew bump-formula-pr` would be nice. This could allow these PRs to be more easily filtered and treated differently by a stale bot (perhaps closed more quickly if their CI status is failing?).

CC @homebrew/core for thoughts and volunteers.

### PR DESCRIPTION
**Pull request by Rylan12**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Closes #76763

Add `bump-formula-pr` label to PRs with the following text (included here so that the label will be applied to this PR - hopefully):

> Created with `brew bump-formula-pr`.

I also changed the indentation in the workflow file to be 2 spaces (which looks better for YAML and conforms with all the other YAML files in the organization). That's done in a separate commit to make reviewing easier.

### CODE REVIEW

**Comment by Rylan12** [comment]
Looks like this works: https://github.com/Homebrew/homebrew-core/pull/76886

**Comment by dawidd6** [comment]
Can we have this in label-pull-requests Action instead of a JS script in YAML?

**Comment by SMillerDev** [comment]
Agree with David here. That can do the same thing for casks in one go.

**Comment by Rylan12** [comment]
Yeah, sure. Okay to just put this in the `label-pull-requests` action that already exists? This doesn't really line up with the other labels that applies because it's checking the PR body rather than the files changed, but probably still makes sense to go there.

**Comment by Rylan12** [comment]
PR opened: https://github.com/Homebrew/actions/pull/174

File: .github/workflows/triage.yml [hunk 1]
```diff
@@ -3,123 +3,126 @@ name: Triage tasks
...
+                      "label": "swift",
+                      "path": "Formula/.+",
+                      "content": "system \"swift\", \"build\""
+                  }, {
+                      "label": "automerge-skip",
+                      "path": "Formula/(patchelf|binutils).rb",
+                      "keep_if_no_match": true
+                  }, {
+                      "label": "bump-formula-pr",
+                      "pr_body_contents": "Created with `brew bump-formula-pr`"
```
Comment (dawidd6) [inline .github/workflows/triage.yml:126]:
```suggestion
                      "pr_body_content": "Created with `brew bump-formula-pr`" 
```

**Review by MikeMcQuaid — DISMISSED** [review]
🎉 very nice!

**Comment by Rylan12** [comment]
Oops, I meant to rebase & merge not squash & merge to keep the indentation change commit separate 🤦 ah well...

**Pull request merged**